charts/harvester-csi-driver-lvm/Chart.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v2
+name: harvester-csi-driver-lvm
+description: A Helm chart for Harvester LVM CSI Driver
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.0.0-dev
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: v0.1.0
```

Comment:
Nit: I'd suggest putting this in quotes (some of our other charts do use quotes, some don't, but https://helm.sh/docs/topics/charts/#the-appversion-field says quotes are recommended)

Reply:
Thanks @tserong.
I will add quotes in the next update.

I would like to have the first release for developing the addon capability.